spec/takeaway_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new }
+  let(:menu) {double('menu', :dishes => { salad: 12 }) }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)